### ISSUE TITLE
The package org.xml.sax is accessible from more than one module: <unnamed>, java.xml

### ISSUE THREAD

**Issue by vogella** [eclipse/windowbuilder#98]
Trying to make the test run, I see the following errors:

The package org.xml.sax is accessible from more than one module: <unnamed>, java.xml 

For example:

[ERROR] The package org.xml.sax is accessible from more than one module: <unnamed>, java.xml
[ERROR] C:\Users\vogella\git\windowbuilder\org.eclipse.wb.tests\src\org\eclipse\wb\tests\designer\core\model\description\ComponentDescriptionHelperTest.java:[845]
[ERROR]         assertInstanceOf(SAXParseException.class, e);
[ERROR]                          ^^^^^^^^^^^^^^^^^
[ERROR] SAXParseException cannot be resolved to a type

**Comment by vogella** [comment]
Strange, no idea what causes this. Maybe the update of the BREE to Java 11 in the text module.

**Comment by vogella** [comment]
Looks like as soon as I increase the BREE in a plug-in to Java 11 and this plug-in uses org.xml.sax  I get this error. 

[ERROR] The package org.xml.sax is accessible from more than one module: <unnamed>, java.xml

**Comment by wimjongman** [comment]
Yes, I think it means that you can remove the package dependency because java.xml is already provided by the jvm
